cli/src/infragpt/llm/providers/openai_provider.py

```
@@ -0,0 +1,241 @@
+            params = {
+                "model": self.model,
+                "messages": [{"role": "user", "content": "hi"}]
+            }
+            
+            # Newer models like o4-mini use max_completion_tokens
+            if self.model.startswith("o4") or self.model.startswith("o1"):
+                params["max_completion_tokens"] = 10
+            else:
+                params["max_tokens"] = 10
+                
+            response = self._client.chat.completions.create(**params)
```

Ruff (0.12.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)